### PR TITLE
[Merged by Bors] - update README of local_testnet

### PR DESCRIPTION
**Pull request by int88**
## Issue Addressed

NA

## Proposed Changes

update the descriptions of README in `scripts/local_testnet`.

## Additional Info

NA

### CODE REVIEW

**Comment by int88** [comment]
@michaelsproul PTAL :)

File: scripts/local_testnet/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Modify `vars.env` as desired.
 Start a local eth1 ganache server plus boot node along with `BN_COUNT`
 number of beacon nodes and `VC_COUNT` validator clients.
 
-The `start_local_testnet.sh` script takes three options `-v VC_COUNT`, `-d DEBUG_LEVEL` and `-h` for help.
+The `start_local_testnet.sh` script takes four options `-v VC_COUNT`, `-d DEBUG_LEVEL`, `-p` to enable private tx proposals and `-h` for help.
```
Comment (michaelsproul) [inline scripts/local_testnet/README.md]:
```suggestion
The `start_local_testnet.sh` script takes four options `-v VC_COUNT`, `-d DEBUG_LEVEL`, `-p` to enable builder proposals and `-h` for help.
```

The "private tx" thing is a bit outdated these days, we tend to refer to this as "builder proposals". You could update the docs in `start_local_testnet.sh` as well if you like

**Review by michaelsproul — CHANGES_REQUESTED** [review]

File: scripts/local_testnet/README.md (hunk 1 above)
Comment (int88) [reply scripts/local_testnet/README.md]: @michaelsproul fixed.

**Review by michaelsproul — APPROVED** [review]

**Comment by int88** [comment]
@michaelsproul  maybe the flaky test should be retriggered.

**Comment by michaelsproul** [comment]
it'll re-run when bors runs, now

bors r+

**Pull request closed without merging**